ssh.bash

```diff
@@ -15,21 +15,36 @@ ssh_read_data() {
   read -r ssh_password < <(echo "$contents")
 
   while read -r line; do
-  if [[ "$line" == ssh:* ]]; then
-    local ssh_address=${line#"ssh:"}
-    ssh_address=${ssh_address##+([[:space:]])}
-  fi
-  if [[ "$line" == sshflags:* ]]; then
-    local ssh_flags=${line#"sshflags:"}
-    ssh_flags=${ssh_flags##+([[:space:]])}
-  fi
+    if [[ "$line" == ssh:* ]]; then
+      local ssh_address=${line#"ssh:"}
+      ssh_address=${ssh_address##+([[:space:]])}
+    fi
+    if [[ "$line" == sshflags:* ]]; then
+      local ssh_flags=${line#"sshflags:"}
+      ssh_flags=${ssh_flags##+([[:space:]])}
+    fi
+    if [[ "$line" == user:* ]]; then
+      local ssh_user=${line#"user:"}
+#      ssh_user=${ssh_user##+([[:space::]])}
+    fi
+    if [[ "$line": == IP:* ]]; then
+      local ssh_host=${line#"IP:"}
+#      ssh_host=${ssh_host##+([[:space::]])}
+    fi
   done < <(echo "$contents")
+  if [[ -z ${ssh_address+x} ]]; then
+    ssh_address="${ssh_user// /}@${ssh_host// /}"
```

Comment:
If `ssh:`, `IP:` and `user:` aren't specified this will effectively disable the empty check in line 39 by setting ssh_address to "@", right?

Reply:
Right, I guess I'll have to do something else.